test/onnx/gen_onnx.py

```diff
@@ -5992,14 +5992,22 @@ def int4_const_identity_qdq_test():
     t_node = helper.make_node(
         'Transpose',
         inputs=['dq_y'],
-        outputs=['y'],
+        outputs=['t_y'],
         perm=[1, 0],
     )
 
+    x2_t = helper.make_tensor_value_info('x2', TensorProto.FLOAT16, [2, 4])
+
+    dot_node = helper.make_node(
+        'MatMul',
+        inputs=['t_y', 'x2'],
+        outputs=['y'],
+    )
```

Comment:
The onnx files are not even being tested. Either way this should go into another PR.

Reply:
These files are already being used, at least one of them. And we can add additional tests later -- or now. Thanks.